.github/PULL_REQUEST_TEMPLATE/bug_template.md

```diff
@@ -1,4 +1,13 @@
-### What does this PR do?
```

Comment:
Hey @floralan Just so I'm clear on the end behavior here.  Will we now have to pick what kind of PR we are opening when opening a new PR through the Github UI?

Reply:
@gbockus-sf Since we don't have the ability to change PR title or having multiple templates, I included the three different PR-title templates into this one template file so people just keep the one they need and delete others.